nvml/manifest.json

```diff
@@ -21,9 +21,9 @@
     ]
   },
   "author": {
-    "support_email": "help@datadoghq.com",
+    "support_email": "cep221@gmail.com",
     "homepage": "https://github.com/DataDog/integrations-extras",
-    "sales_email": "help@datadoghq.com",
+    "sales_email": "cep221@gmail.com",
```

Comment:
We should just deprecate and point people towards https://www.datadoghq.com/blog/monitor-nvidia-gpus-with-datadog/#start-monitoring-your-gpu-powered-workloads